app/services/team_importer.rb

```diff
@@ -642,7 +642,6 @@ def create_experiment(experiment_json, project, user_id = nil)
       user_id || find_user(experiment.last_modified_by_id)
     experiment.archived_by_id = find_user(experiment.archived_by_id)
     experiment.restored_by_id = find_user(experiment.restored_by_id)
-    experiment.skip_user_assignments = true
```

Comment:
I think then create_user_assignments will fail on line 649. We have validation for assignments uniqness

Reply:
Same for my my_module

Reply:
But don't we create a new set of user assignments for a new object?

Reply:
In case of team import, we need to skip it. We export all assignments from the original team, they are part of JSON.

Reply:
ok